clients/src/main/java/org/apache/kafka/common/config/TopicConfig.java

```diff
@@ -65,7 +65,9 @@
     public static final String RETENTION_BYTES_CONFIG = "retention.bytes";
     public static final String RETENTION_BYTES_DOC = "This configuration controls the maximum size a log can grow " +
         "to before we will discard old log segments to free up space if we are using the " +
-        "\"delete\" retention policy. By default there is no size limit only a time limit.";
+        "\"delete\" retention policy. By default there is no size limit only a time limit." +
+        " Please note, this is calculated as retention.bytes * number of partitions on the given topic" +
+        " for the total amount of disk space to be used.";
```

Comment:
I am not sure about the additional sentence. It's not quite accurate since it depends on the size of the active log segment.

Reply:
hmm.. My idea is to give a rough estimate of total topic disk space to be used. Do you have any suggestion for this doc sentence?